src/validators/create-todo-validator.ts

```diff
@@ -0,0 +1,19 @@
+import lodash from 'lodash';
+import { AppContext } from '@typings';
+import { check, ValidationChain } from 'express-validator';
+
+const createTodoValidator = (appContext: AppContext): ValidationChain[] => [
+  check('title', 'VALIDATION_ERRORS.INVALID_TITLE').notEmpty(),
+  check('title').custom(async (title) => {
+    const todo = await appContext.todoRepository.findOne({
+      title
+    });
+
+    if (!lodash.isEmpty(todo)) {
+      return Promise.reject();
+    }
+  })
+  .withMessage('VALIDATION_ERRORS.DUPLICATE_ENTRY')
+];
+
+export default createTodoValidator;
```

Comment:
Please do make sure you are leaving a new line by EOF for all files which are created (You can use EditorConfig - https://editorconfig.org/ which would do this automatically for you)